README.md

```diff
@@ -259,6 +258,74 @@ curl http://aws-java-sample-java-function.default.dev.triggermesh.io -d '{"event
+
+- [go](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/go-1.x/runtime.yaml#L44-L68)
+- [java](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/java8/runtime.yaml#L43-L53)
+- [node-10](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/node-10.x/runtime.yaml#L43-L48)
+- [node-4](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/node-4.x/runtime.yaml#L43-L48)
+- [python-2](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-2.7/runtime.yaml#L43-L50)
+- [python-3](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-3.7/runtime.yaml#L43-L50)
+- [ruby-2](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/ruby-2.5/runtime.yaml#L43-L47)
+
+
+Let's build Python-3.7 function as an example:
+
+1. Create directory and save your function code there:
+
+```
+  mkdir python
+  cd python
+  cat > handler.py <<EOF
+import json
+import datetime
+def endpoint(event, context):
+  current_time = datetime.datetime.now().time()
+  body = {
+      "message": "Hello, the current time is " + str(current_time)
+  }
+  response = {
+      "statusCode": 200,
+      "body": json.dumps(body)
+  }
+  return response
+EOF
+```
+
+2. Extract runtime [Dockerfile](https://github.com/triggermesh/knative-lambda-runtime/blob/9a74ce1ac03d56d233cfc7a46d84f2c5e5f2685a/python-3.7/runtime.yaml#L43-L50), update `_HANDLER` variable and store it in the same directory:
+
+```
+  cat > Dockerfile <<EOF
+  FROM gcr.io/triggermesh/knative-lambda-python37
+  ENV _HANDLER handler.endpoint
+  COPY . .
+  RUN if [ -f requirements.txt ]; then pip3.7 install -r requirements.txt ;fi
+  ENTRYPOINT ["/opt/aws-custom-runtime"]
+EOF
+```
+
+`_HANDLER` variable in most cases consist of the filename without the file format part and function name.
+
+3. Build, run, test:
+
+```
+  docker build -t python-klr-image .
+  docker run -d --rm --name python-klr-container python-klr-image
+  # following command will work if you use Docker bridge network and you have jq tool 
+  # otherwise, you should get the container address manually  
+  curl $(docker inspect python-klr-container | jq .[].NetworkSettings.Networks.bridge.IPAddress -r):8080
+```
+  
+  The response will contain JSON with the current time.
+
+4. Cleanup:
+
+```
+  docker stop python-klr-container
```

Comment:
and docker kill

Reply:
I thought that `stop` is `SIGTERM` -> 10s wait -> `SIGKILL`, while `kill` is just `SIGKILL`, but I may be wrong :thinking:

Reply:
Agree, please never use kill unless absolutely necessary.

I think Seb meant `rm`.

Reply:
yes my bad. I meant `rm`

Reply:
I knew that you'll ask about the `rm` and that's why I put it in the `run` command: 
```
docker run -d --rm --name python-klr-container python-klr-image
```
https://github.com/triggermesh/knative-lambda-runtime/pull/50/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R315

> --rm                             Automatically remove the container when it exits